test/cpp/server/credentials_test.cc

```diff
@@ -123,6 +153,73 @@ TEST(
   GPR_ASSERT(server_credentials.get() != nullptr);
 }
 
+TEST_F(
+    GrpcTlsCertificateKeyMatchTest,
+    APIWrapperForPrivateKeyAndCertificateMatchForFailedKeyCertMatchOnEmptyPrivateKey){
```

Comment:
Let's change the name to `CoreAPICertificateKeyMatchFailedOnEmptyKey`. Similar for other tests.